keyboards/primekb/prime_l/v1/readme.md

```diff
@@ -0,0 +1,15 @@
+# Prime_L
+
+![Prime_L](https://i.imgur.com/jUWhjOD.jpg)
+
+A compact layout with the footprint of your typical 65% board, featuring a left mini-numpad. Pre-drilled and counter sunk holes for use with optional feet for a slight tilt. Switches are oriented so that back light LEDs are on the top of the switch.
+
+Keyboard Maintainer: [MxBlue](https://github.com/MxBlu)  
+Hardware Supported: Prime_L PCBs, ATMega32u4  
+Hardware Availability: [Store Link](https://www.primekb.com/collections/keyboards/products/bundle-save)
```

Comment:
This store link is Error 404. You should provide an updated one, or some sort of reference page if there's one available. Otherwise this should note the V1 is no longer available.